users/api/http/transport.go

```diff
@@ -53,13 +56,34 @@ func MakeHandler(svc users.Service, tracer opentracing.Tracer, l log.Logger) htt
 		opts...,
 	))
 
+	mux.Post("/password/request", kithttp.NewServer(
```

Comment:
Can you use `/password/reset-request` or something like that? I think that it's not clear from this path that this will start password reset flow.